_includes/filter-tabs.md

```diff
@@ -0,0 +1,15 @@
+{% comment %}
+In order to properly utilize this file, the following three items must be added to the YAML front matter of each page:
+
+filter_category: The name of the group of pages where the filter will be applied. This can be anything you want, but all pages that utilize the same filter must be set to the same filter_category.
+filter_html: The HTML of the text to be used for that page's tab in the filter.
+filter_sort: The sort order of that particular page's tab in the filter.
```

Comment:
If two pages were accidentally assigned the same sort order, the pages would be sorted based on their file name. For example, the [GO tutorials](https://deploy-preview-12621--cockroachdb-docs.netlify.app/docs/v21.2/build-a-go-app-with-cockroachdb) would order it in GORM, pq, upper/db, and then pgx (because period is sorted after hyphens).